pkg/mycarehub/usecases/user/user.go

```diff
@@ -965,6 +965,25 @@ func (us *UseCasesUserImpl) RegisterExistingUserAsClient(ctx context.Context, in
 		return nil, err
 	}
 
+	cmsUserPayload := &dto.PubsubCreateCMSClientPayload{
+		ClientID: *registeredClient.ID,
+		Name:     registeredClient.User.Name,
+		Gender:   registeredClient.User.Gender.String(),
+		DateOfBirth: scalarutils.Date{
+			Year:  registeredClient.User.DateOfBirth.Year(),
+			Month: int(registeredClient.User.DateOfBirth.Month()),
+			Day:   registeredClient.User.DateOfBirth.Day(),
+		},
+		OrganisationID: registeredClient.OrganisationID,
+		ProgramID:      registeredClient.ProgramID,
+	}
+
+	err = us.Pubsub.NotifyCreateCMSClient(ctx, cmsUserPayload)
+	if err != nil {
+		helpers.ReportErrorToSentry(err)
+		log.Printf("filed to publish client profile to cms: %v", err)
```

Comment:
Typo here
```suggestion
		log.Printf("failed to publish client profile to cms: %v", err)
```

Reply:
I'm also expecting the tests to fail for this implementation. Especially the sad case

Reply:
No error is being thrown since it is a pub sub-tasks. If it fails, it will automatically retry without affecting the rest of the program. The error will be logged to sentry